README (2).md

```diff
@@ -0,0 +1,49 @@
+![PollCord_Logo](https://user-images.githubusercontent.com/36363608/193755078-ac3949c0-0db9-4697-bf99-cc62e3788621.png)
+
+
+[![MIT License](https://img.shields.io/badge/License-MIT-green.svg)](https://choosealicense.com/licenses/mit/)
+
+[![Python](https://img.shields.io/badge/python-v3.7+-green.svg)](https://docs.python.org/3/)
+#    PollCord
+
+### What is PollCord?
+Pollcord is a software created for Disocrd users to conduct Polls
+
+### Why PollCord?
+Disocrd is a social platform where users interact with each other. Creating Polls is a sophisticated way to ask questions, collect opinions especially with large number of users. However, Disocrd currently lacks this feature to create user-friendly interactive Polls. PollCord is an attempt to bridge this gap
```

Comment:
Discord does have several interactive poll-like features that can be installed in the form of bots. However, few points you can say to make PollCord stand out:
- PollCord can aggregate multiple questions on a single poll rather than just one poll at a time, and allows for multiple people or just the poll creator to manage the questions and their options
- Using NextCord, which is a cutting edge API wrapper for Discord, to create a modern interactive experience compared to most other Discord poll bots